pom.xml

```diff
@@ -110,6 +110,11 @@ under the License.
       <artifactId>javax.inject</artifactId>
       <version>1</version>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.sisu</groupId>
+      <artifactId>org.eclipse.sisu.plexus</artifactId>
+      <scope>provided</scope>
+    </dependency>
```

Comment:
Is it needed?

Reply:
Yes, according to mvn dependency:analyze:

```
[INFO] --- dependency:3.6.1:analyze (default-cli) @ maven-rar-plugin ---
[WARNING] Used undeclared dependencies found:
[WARNING]    org.eclipse.sisu:org.eclipse.sisu.plexus:jar:0.9.0.M2:provided
[WARNING] Unused declared dependencies found:
[WARNING]    org.apache.maven:maven-settings:jar:3.6.3:provided
[WARNING]    org.codehaus.plexus:plexus-interpolation:jar:1.27:compile
[WARNING]    org.codehaus.plexus:plexus-xml:jar:3.0.0:compile
[WARNING]    junit:junit:jar:4.13.2:test
[WARNING]    org.apache.maven:maven-compat:jar:3.6.3:test
[INFO] ------------------------------------------------------------------------
```